src/lib/types.ts

```diff
@@ -106,5 +108,12 @@ export type Bid = {
 	amount: number;
 	bidderAccount: string;
 	date: Date;
-	tokenId?: string;
+	tokenId: number;
+
+	signature: ethers.utils.BytesLike;
+	auctionId: number;
+	nftAddress: string;
+	minBid: number;
```

Comment:
minBid, startBlock, and expireBlock are all potentially uint256's, so keep these as strings please